src/code/code_8006C3A0.c

```diff
@@ -1,30 +1,30 @@
 #include "global.h"
 
-void Flags_UnsetAllEnv(PlayState* play) {
+void CutsceneFlags_UnsetAll(PlayState* play) {
     u8 i;
 
     for (i = 0; i < ARRAY_COUNT(play->envFlags); i++) {
         play->envFlags[i] = 0;
```

Comment:
If the system is `CutsceneFlags`, should the struct member change from `envFlags`?

Reply:
yea i guess i just forgot lol